packages/env/package.json

```diff
@@ -39,6 +39,7 @@
 		"got": "^10.7.0",
 		"inquirer": "^7.1.0",
 		"js-yaml": "^3.13.1",
+		"lodash": "^4.17.15",
```

Comment:
We can use `Object.keys`, `Object.values`, and `Object.entries` to avoid using Lodash.